dev-support/jenkins-scripts/cache-apache-project-artifact.sh

```diff
@@ -123,6 +123,12 @@ if [ -n "${keys}" ]; then
   echo "signature good."
 fi
 
+if [ -n "${keys}" ]; then
```

Comment:
this should be placed up in the clean up handler function. that way if e.g. we run into a problem before here but after our first command launches a gpg-agent we'll still clean up.

Reply:
@busbey Thanks for the review. Addressed you review comment in latest commit. Please review again.